### PR TITLE
Synchronize zaaktypen

### PR DESCRIPTION
**Pull request by rjzondervan**

### CODE REVIEW

File: Installation/Source/requestInbox.source.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  "auth": "oauth",
+  "authenticationConfig": {
+    "tokenUrl": "https://request-inbox.vrijbrp.nl/auth",
+    "tokenField": "token"
+  },
+  "authorizationPassthroughMethod": "base_auth",
+  "username": "!ChangeMe!",
+  "password": "!ChangeMe!",
+  "documentation": null,
+  "loggingConfig": {
```
Comment (WilcoLouwerse) [inline Installation/Source/requestInbox.source.json:20]: These are default values, you should not need to set these

File: Installation/Source/requestInbox.source.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    "callContentType": true,
+    "callBody": true,
+    "responseStatusCode": true,
+    "responseContentType": true,
+    "responseBody": true,
+    "maxCharCountBody": 500,
+    "maxCharCountErrorBody": 2000
+  },
+  "configuration": {
+    "verify": true
```
Comment (WilcoLouwerse) [inline Installation/Source/requestInbox.source.json:33]: These are default values, you should not need to set these

**Review by WilcoLouwerse — APPROVED** [review]
How did you test this without Commando or Endpoint?

File: Installation/Source/requestInbox.source.json (hunk 1 above)
Comment (rjzondervan) [reply Installation/Source/requestInbox.source.json:20]: I know, but it is nice to have them for reference, as you can't set these correctly via the ui

File: Installation/Source/requestInbox.source.json (hunk 2 above)
Comment (rjzondervan) [reply Installation/Source/requestInbox.source.json:33]: I like to have it in case I need to add more config

**Comment by rjzondervan** [comment]
This was tested via a trigger, it should be run via a cronjob eventually.

**Pull request merged**